src/app/organization/employees/create-employee/create-employee.component.ts

```diff
@@ -0,0 +1,91 @@
+/** Angular Imports */
+import { Component, OnInit } from '@angular/core';
+import { FormGroup, FormBuilder, Validators } from '@angular/forms';
+import { Router, ActivatedRoute } from '@angular/router';
+
+/** Custom Services */
+import { OrganizationService } from '../../organization.service';
+
+/**
+ * Create employee component.
+ */
+@Component({
+  selector: 'mifosx-create-employee',
+  templateUrl: './create-employee.component.html',
+  styleUrls: ['./create-employee.component.scss']
+})
+export class CreateEmployeeComponent implements OnInit {
+
+  /** Minimum joining date allowed. */
+  minDate = new Date(2000, 0, 1);
+  /** Maximum joining date allowed. */
+  maxDate = new Date();
+  /** Employee form. */
+  employeeForm: FormGroup;
+  /** Office data. */
+  officeData: any;
+
+  /**
+   * Retrieves the offices data from `resolve`.
+   * @param {FormBuilder} formBuilder Form Builder.
+   * @param {OrganizationService} organizationService Organization Service.
+   * @param {ActivatedRoute} route Activated Route.
+   * @param {Router} router Router for navigation.
+   */
+  constructor(private formBuilder: FormBuilder,
+              private organizationService: OrganizationService,
+              private route: ActivatedRoute,
+              private router: Router) {
+    this.route.data.subscribe((data: { offices: any }) => {
+      this.officeData = data.offices;
+    });
+  }
+
+  /**
+   * Creates the employee form.
+   */
+  ngOnInit() {
+    this.createEmployeeForm();
+  }
+
+  /**
+   * Creates the employee form.
+   */
+  createEmployeeForm() {
+    this.employeeForm = this.formBuilder.group({
+      'officeId': ['', Validators.required],
+      'firstname': ['', Validators.required],
+      'lastname': ['', Validators.required],
+      'isLoanOfficer': [false, Validators.required],
```

Comment:
```suggestion
      'isLoanOfficer': [false],
```
Validator not required for this.